packages/css-variables/test/test.js

```diff
@@ -0,0 +1,33 @@
+import 'babel-polyfill'
+
+import toCssVariables from '../src'
+
+const theme = {
+  colors: {
+    text: '#000',
+    background: '#fff',
+    primary: '#07c',
+    secondary: '#05a',
+    accent: '#609',
+    muted: '#f6f6f6',
```

Comment:
Don't worry about it for this PR, but support for setting colors modes via a class selector is something I've thought might be nice in a utility like this

Reply:
That would be a nice feature. I will create an issue for this after the PR is merged.

Reply:
New issue created https://github.com/system-ui/theme-ui/issues/257